### DISCUSSION TITLE
When specifying 'or' as the query parameter the sql shows an 'and' clause, why is this?

### DISCUSSION THREAD

**Discussion by atreeon** [PostgREST/postgrest#2463] [Q&A] [answered]
My get request is this
`employees?employee_id=gt.4&or=(employee_id.eq.6)`

but the sql (using supabase logging) is the following
`SELECT "public"."employees".*FROM "public"."employees"  WHERE  ( "public"."employees"."employee_id" = $1) AND  "public"."employees"."employee_id" > $2`

I would have expected the AND to be an OR.

This of course produces an unexpected result.  Am I misunderstanding how postgrest works?

(full sql is the following)
`WITH pgrst_source AS ( SELECT "public"."employees".*FROM "public"."employees"  WHERE  ( "public"."employees"."employee_id" = $1) AND  "public"."employees"."employee_id" > $2  LIMIT $3 OFFSET $4 )  SELECT null::bigint AS total_result_set, pg_catalog.count(_postgrest_t) AS page_total, array[]::text[] AS header, coalesce(json_agg(_postgrest_t), '[]')::character varying AS body, nullif(current_setting('response.headers', true), '') AS response_headers, nullif(current_setting('response.status', true), '') AS response_status FROM ( SELECT * FROM pgrst_source ) _postgrest_t`

**Comment by steve-chavez** [comment, marked as answer]
You need to put it as:

```
employees?or=(employee_id.eq.6,employee_id=gt.4)
```

Check https://postgrest.org/en/stable/api.html#logical-operators

> Multiple conditions on columns are evaluated using AND by default

That applies because your `or` has a single value

**Comment by atreeon** [comment]
I got it! Thanks
